app/points/domain/workload-owner.js

```diff
@@ -0,0 +1,19 @@
+const assertNumber = require('./validation/assert-number')
+
+class WorkloadOwner {
+  constructor (id, offenderManagerId, workingHoursId, teamId) {
+    this.id = id
+    this.offenderManagerId = offenderManagerId
+    this.workingHoursId = workingHoursId
+    this.teamId = teamId
+    this.isValid()
+  }
+
+  isValid () {
+    assertNumber(this.id, 'id')
```

Comment:
Dependent on the use case `id` may not be required. For example, if we are using this object to pass to knex to do an insert we won't yet have an `id`. If the use case is that we parse data from the `app` schema into these objects when we read them then the ID will indeed be required.